### PR TITLE
add job e2e

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor <1187526662@qq.com>

### CODE REVIEW

File: test/e2e/admission.go [hunk 1]
```diff
@@ -220,4 +220,1103 @@ var _ = ginkgo.Describe("Job E2E Test: Test Admission service", func() {
 		_, err = ctx.kubeclient.CoreV1().Pods(ctx.namespace).Create(pod)
 		gomega.Expect(err.Error()).Should(gomega.ContainSubstring(`the podgroup phase is Pending`))
 	})
+
+	ginkgo.It("Job mutate check", func() {
+		ctx := initTestContext(options{})
+		defer cleanupTestContext(ctx)
+
+		var job v1alpha1.Job
+		jsonData := []byte(`{
```
Comment (hzxuzhonghu) [inline test/e2e/admission.go:229]: Why not define Job directly here?

File: test/e2e/admission.go [hunk 2]
```diff
@@ -220,4 +220,1103 @@ var _ = ginkgo.Describe("Job E2E Test: Test Admission service", func() {
...
+			 "ttlSecondsAfterFinished": -1
+		 }
+	 }`)
+		err := json.Unmarshal(jsonData, &job)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		_, err = ctx.vcclient.BatchV1alpha1().Jobs(ctx.namespace).Create(&job)
+		gomega.Expect(err).To(gomega.HaveOccurred())
+	})
+
+	ginkgo.It("job validate check: illegal minAvailable when create", func() {
```
Comment (hzxuzhonghu) [inline test/e2e/admission.go:542]: `minAvailable`？

File: test/e2e/admission.go (hunk 1 above)
Comment (hzxuzhonghu) [reply test/e2e/admission.go:229]: Applies to all cases

Comment (Thor-wl) [reply test/e2e/admission.go:229]: black box test use json style input may be better?

File: test/e2e/admission.go (hunk 2 above)
Comment (Thor-wl) [reply test/e2e/admission.go]: It is strange that no such bug in my local workspace, and neither is encode mistake. I've modified the comments

**Comment by hzxuzhonghu** [comment]
/lgtm

Need to migrate some cases to ut to reduce the time running.

**Pull request merged**